docs/apis-tools/spring-zeebe-sdk/configuration.md

```diff
@@ -11,6 +11,62 @@ Configuration properties can be defined as environment variables using [Spring B
 For example, the property `camunda.client.zeebe.defaults.max-jobs-active` is represented by the environment variable `CAMUNDA_CLIENT_ZEEBE_DEFAULTS_MAXJOBSACTIVE`.
 :::
 
+## Modes
+
+The Spring SDK has modes with meaningful defaults aligned with the distribution's default connection details.
+
+:::note
+The defaults applied by the modes are overwritten by _any_ other set property, including legacy/deprecated properties. Check your configuration and logs to avoid unwanted override.
+:::
+
+### SaaS
+
+Activate by setting:
+
+```yaml
+camunda:
+  client:
+    mode: saas
+```
+
+This applies the following defaults:
+
+```yaml reference referenceLinkText="Source" title="SaaS mode"
+https://github.com/camunda/camunda/blob/main/clients/spring-boot-starter-camunda-sdk/src/main/resources/modes/saas.yaml
+```
+
+### Self-Managed
+
+Activate by setting:
+
+```yaml
+camunda:
+  client:
+    mode: self-managed
+```
+
+This applies the following defaults:
+
+```yaml reference referenceLinkText="Source" title="Self-managed mode"
+https://github.com/camunda/camunda/blob/main/clients/spring-boot-starter-camunda-sdk/src/main/resources/modes/self-managed.yaml
+```
+
+### Basic
```

Comment:
What does the basic mode mean? Can you only have one mode set at a time?

Reply:
Yes. With 8.8, there will be a basic mode where authentication to the API happens with username and password instead of JWT.
The basic mode will configure the client to use this way of authentication.

Reply:
Should this be refined to become more explicit?

Reply:
Up to you -- as a user who wasn't familiar with this topic they were just questions I had top of mind while reading through the docs for the first time.

Reply:
Very good feedback. I will add:
* the information that only one mode can be used at a time
* outline what their purpose is

Reply:
DOne